types/dayjs-precise-range/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for dayjs-precise-range 1.0
+// Project: https://github.com/huangjinlin/dayjs-precise-range
+// Definitions by: Eugene Shangin <https://github.com/fessracha>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
```

Comment:
```suggestion
```
It's no longer suppoert on DT (3.4 is the latest IMO), this should be used only if specific language features are required